docs/UserGuide.md

```diff
@@ -487,6 +487,14 @@ Format: `exit`
 
 --------------------------------------------------------------------------------------------------------------------
 
+## Glossary
+
+* **Business Contact**: Synonymous with **Client**
+* **Client**: Refers to a person whom the user is conducting his/her business with
+* **GMT**: [Greenwich Mean Time](https://en.wikipedia.org/wiki/Greenwich_Mean_Time)
+* **Mainstream OS**: Windows, Linux, Unix, OS-X
```

Comment:
Is it possible to change OS-X to macOS for both UG and DG? I think it'll be better

Reply:
Yup agree, made the changes